Configuration/src/CloudFoundry/CloudFoundry.feature

```diff
@@ -4,17 +4,17 @@ Feature: CloudFoundry Configuration
   You can run some CloudFoundry configuration samples
 
   @net6.0
-  @win10-x64
-  Scenario: CloudFoundry Configuration (net6.0/win10-x64)
+  @win10
+  Scenario: CloudFoundry Configuration (net6.0/win10)
```

Comment:
I was thinking about the [available RIDs ](https://learn.microsoft.com/en-us/dotnet/core/rid-catalog#using-rids) the other day when I was looking at these, but this change inspired me to go see if anything has changed... it looks like we could do `-r win-x64` and drop the `10`, but I don't know that it would have any benefit any time soon

Reply:
The text here, IIRC, was to match the "@tag", e.g. "@linux-x64".  Seems that the "-x64" part will be superfluous for the near future so just dropped it.  "x64" is the new "x32".